src/legacy/core_plugins/data/public/index.scss

```
@@ -1,5 +1,3 @@
 @import 'src/legacy/ui/public/styles/styling_constants';
 
-@import './query/query_bar/index';
-
-@import '../../../../plugins/data/public/ui/index'
\ No newline at end of file
+@import '../../../../plugins/data/public/ui/index'
```

nit: design team were requesting me to import from the root of the plugin in the plugins I migrated, i.e.

```suggestion
@import '../../../../plugins/data/public/index'
```

seems they missed it in this PR. Maybe we could do that for consistency&mdash;every plugin has `index.scss` in the root.